libs/api/src/lib/graphql/mutation.public.ts

```diff
@@ -421,7 +427,8 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
             saveCustomer: true,
             paymentMethodID: paymentMethod.id,
             successURL,
-            failureURL
+            failureURL,
+            migrateToTargetPMid: memberPlan.migrateToTargetPMid
```

Comment:
Why is this needed in the registerMemberAndReceivePayment function should here not be the payment methode already correct since its a new subscription?

Reply:
one could argue so. probably the expected behavior is more concise. 

:heavy_check_mark: done